.travis.yml

```diff
@@ -5,9 +5,13 @@ go:
 
 services:
   - docker
+  - postgresql
 
 sudo: false
 
+before_script:
+  - psql -c 'create database docker_test;' -U postgres
+
```

Comment:
this is running in travis so not worried about that.

the creation of the databases is a prerequisite to running the application and not in the purview of the migrations themselves. 

well at least i don't think it should be, but i am happy to be convinced.